### PR TITLE
chore: adding regex questions

### PR DESCRIPTION
**Pull request by sahilbheke**
added one question.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/regex-quiz.ts
Comment (jdwilkin4) [inline src/data/regex-quiz.ts:28]: both of these questions already exist in the quiz file.
Please remove both of them

File: src/data/regex-quiz.ts
Comment (jdwilkin4) [inline src/data/regex-quiz.ts:7]:
```suggestion
    Distractor1: "Any number",
    Distractor2: "Any special characters",
    Distractor3: "Only upper case letters,
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/regex-quiz.ts
Comment (sahilbheke) [reply src/data/regex-quiz.ts:28]: Yeah sure, Can you please hacktoberfest-accepted label to my PR? Because the first two contributions were accepted by hacktoberfest  but not the last two.

Comment (jdwilkin4) [reply src/data/regex-quiz.ts]: @sahilbheke 

>Can you please hacktoberfest-accepted label to my PR?

So the hacktoberfest accepted label is applied after a maintainer approves the PR.
So this PR will get that label after I approve and will count towards your total.

>but not the last two

The last two PRs here were closed because they were duplicate questions and will not be accepted into the project.
https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/759
https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/758

Those PR's will not get the hacktoberfest accepted label because they were not approved and will not go into the main branch.

Here is the official Hacktoberfest rules found on the [participation page](https://hacktoberfest.com/participation/)

Project maintainers must accept your pull/merge requests for them to count toward your total

Hope that helps 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

File: src/data/regex-quiz.ts
Comment (sahilbheke) [reply src/data/regex-quiz.ts]: Thank you for helping.

**Pull request merged**
